src/Appwrite/Utopia/Response/Model/Webhook.php

```diff
@@ -49,7 +49,7 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'Webhook trigger events.',
                 'default' => [],
-                'example' => 'database.collections.update',
+                'example' => ['databases.*.collections.*.update'],
```

Comment:
<img width="801" alt="Screen Shot 2022-08-11 at 2 24 17 PM" src="https://user-images.githubusercontent.com/29069505/184211851-b4204c08-46d3-4824-a73c-a240afe1fcff.png">
Again the example works, but the links are escaped weirdly